app/connectors/IvdSubmissionConnector.scala

```diff
@@ -34,6 +34,8 @@ class IvdSubmissionConnector @Inject()(val http: HttpClient,
 
   private[connectors] def createCaseUrl = s"${config.importVoluntaryDisclosureSubmission}/api/case"
 
+  private[connectors] def updateCaseUrl = s"${config.importVoluntaryDisclosureSubmission}/api/update-case"
```

Comment:
Is this a def for a specific reason? Doesn't take in arguments so why not a val?